reana/cli.py

```diff
@@ -1047,3 +1059,40 @@ def run_example(component, workflow_engine, output, sleep):  # noqa: D301
                 run_command(cmd, component)
     # report status; everything was OK
     run_command('echo OK', component)
+
+
+@click.option('--component', '-c', multiple=True, default=['ALL'],
+              help='Which components? [shortname|name|.|CLUSTER|ALL]')
+@cli.command(name='kubectl-delete-pod')
+def kubectl_delete_pod(component):  # noqa: D301
+    """Delete REANA component's pod.
+
+    If option ``component`` is not used, all pods will be deleted.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-job-controller';
+                         * (2) short component name such as 'r-j-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :type component: str
+
+    """
+    if "ALL" in component:
+        cmd = 'kubectl delete --all pods'
+        run_command(cmd)
+    else:
+        components = select_components(component)
+        for component in components:
+            if component in COMPONENT_PODS:
+                cmd = 'kubectl delete pod -l app={0}'.format(
+                    COMPONENT_PODS[component])
+                run_command(cmd)
```

Comment:
Please use ` run_command(cmd, component)` so that the output messages wwould be more informative  (will print the leading component in brackets as usual for other commands)